metricbeat/docs/modules/redis.asciidoc

```diff
@@ -24,13 +24,12 @@ Redis has two additional config options:
 
 === Example Configuration
 
-The Apache module supports the standard configuration options which can be found
+The Redis module supports the standard configuration options which can be found
```

Comment:
Will add it. This depends on the YAML file. Not even sure if we should keep Status for the others, as a module can have metricsets that report mor then just status.